examples/README.md

```diff
@@ -1,36 +1,36 @@
 # NVIDIA FLARE Examples
 
-[NVIDIA FLARE](https://nvidia.github.io/NVFlare) provides several examples to help you get started using federated learning for your own applications.
+[NVIDIA FLARE](https://nvflare.readthedocs.io/en/2.1.1/index.html) provides several examples to help you get started using federated learning for your own applications.
 
-The provided examples cover different aspects of [NVIDIA FLARE](https://nvidia.github.io/NVFlare), such as using the provided [Controllers](https://nvidia.github.io/NVFlare/programming_guide/controllers.html) for "scatter and gather" or "cyclic weight transfer" workflows and example [Executors](https://nvidia.github.io/NVFlare/apidocs/nvflare.apis.html?#module-nvflare.apis.executor) to implement your own training and validation pipelines. Some examples use the provided "task data" and "task result" [Filters](https://nvidia.github.io/NVFlare/apidocs/nvflare.apis.html?#module-nvflare.apis.filter) for homomorphic encryption and decryption or differential privacy. Furthermore, we show how to use different components for FL algorithms such as [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [FedOpt](https://arxiv.org/abs/2003.00295). We also provide domain-specific examples for deep learning and medical image analysis.
+The provided examples cover different aspects of [NVIDIA FLARE](https://nvflare.readthedocs.io/en/2.1.1/index.html), such as using the provided [Controllers](https://nvflare.readthedocs.io/en/2.1.1/programming_guide/controllers.html) for "scatter and gather" or "cyclic weight transfer" workflows and example [Executors](https://nvflare.readthedocs.io/en/2.1.1/apidocs/nvflare.apis.html?#module-nvflare.apis.executor) to implement your own training and validation pipelines. Some examples use the provided "task data" and "task result" [Filters](https://nvflare.readthedocs.io/en/2.1.1/apidocs/nvflare.apis.html?#module-nvflare.apis.filter) for homomorphic encryption and decryption or differential privacy. Furthermore, we show how to use different components for FL algorithms such as [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [FedOpt](https://arxiv.org/abs/2003.00295). We also provide domain-specific examples for deep learning and medical image analysis.
```

Comment:
should api related link be: `https://nvflare.readthedocs.io/en/2.1.1/apidocs/nvflare.apis.executor.html`?

Reply:
thanks for noting this, readthedocs builds these pages a bit differently, so these links do need to be updated.